source/Fleet.h

```diff
@@ -82,13 +95,14 @@ class Fleet {
 
 
 private:
+	bool hasConditions = true;
 	std::string fleetName;
 	const Government *government = nullptr;
 	const Phrase *names = nullptr;
 	const Phrase *fighterNames = nullptr;
-	WeightedList<Variant> variants;
+	WeightedList<Variant,WeightType> variants;
 	// The number of different items the ships in this fleet will carry in cargo.
-	int cargo = 3;
+	Condition<int> cargo;
```

Comment:
pretty sure you can just say = 3 or smth of the like here instead of doing it weirdly in the constructor?

Reply:
No, because I had to disable implicit conversion from T to `Condition<T>` to avoid wiping out the condition's key by accident. I can do `Condition<int> cargo = Condition<int>(3)` though, and I just made that change.